stats/release_queue.php

```diff
@@ -150,7 +151,8 @@ function _show_round_queues($round, $listing_view_mode)
         echo "<td>$length</td>\n";
         echo "<td>$unheld_length</td>\n";
         if (user_can_see_queue_settings()) {
-            echo "<td>", html_safe($queue_data["release_criterion"]), "</td>\n";
+            echo "<td>", $queue_data["projects_target"], "</td>\n";
+            echo "<td>", $queue_data["pages_target"], "</td>\n";
         }
```

Comment:
As an alternative to Casey's "display them as a formatted expression", displaying the thresholds with  ≥ and aligning them, might make them read more clearly to someone who hasn't looked at the docs.
You could do something like
`echo "<td class='right-align'> &ge;", number_format($queue-data[...]), "</td>";`